pytorch_blade/tests/torch-disc-pdll/tests/fake_quant.mlir

```diff
@@ -0,0 +1,18 @@
+// RUN: torch-disc-pdll --payload-input %s --pdl-input %p/fake_quant.pdll  | FileCheck %s
+
+// CHECK-LABEL:  func.func @torch_blade_custom_fake_quant
+func.func @torch_blade_custom_fake_quant(%arg0: !torch.tensor, %arg1: !torch.tensor, %arg2: !torch.tensor) -> !torch.tensor {
+  %0 = torch.tensor_static_info_cast %arg0 : !torch.tensor to !torch.tensor<[2,3],f32>
+  %1 = torch.tensor_static_info_cast %arg1 : !torch.tensor to !torch.tensor<[],f32>
+  %2 = torch.tensor_static_info_cast %arg2 : !torch.tensor to !torch.tensor<[],si64>
+  %3 = torch.prim.ListConstruct  : () -> !torch.list<int>
+  %int-127 = torch.constant.int -127
+  %int128 = torch.constant.int 128
+  %int8 = torch.constant.int 8
+  %true = torch.constant.bool true
+  %false = torch.constant.bool false
+  // CHECK: torch.operator
```

Comment:
This check is too general.  It'll pass even though the converter is failed to apply.

better to check `torch_blade.quant/torch_blade.dequant` explicitly and check the `torch_blade.dequant` is the return value.

Reply:
Add more check.